src/Microsoft.AspNet.SignalR.Client/Transports/TransportHelper.cs

```diff
@@ -57,8 +57,6 @@ public virtual Task<string> GetStartResponse(IHttpClient httpClient, IConnection
 
             var startUrl = UrlBuilder.BuildStart(connection, transport, connectionData);
 
-            httpClient.Initialize(connection);
```

Comment:
It seems kinda fragile depending on GetNegotiationResponse to be called immediately before GetStartResponse on the same IHttpClient. I don't think either of the Get* methods should call Initialize if they both can't.

We should call Initialize in Connection.Start(IHttpClient httpClient) instead.